src/components/print/PrintDirective.js

```diff
@@ -399,7 +399,8 @@ goog.require('ga_urlutils_service');
                 }
                 canceller = $q.defer();
                 $http.get(url, {
-                  timeout: canceller.promise
+                  timeout: canceller.promise,
+                  cache: false
```

Comment:
`false` is already  the default value.

Reply:
In any case, the `cache` property is angular internally only, it caches responses in JS scope and doesn't use browsers's cache. Also, it doesn't impact http headers at all, see [doc](https://docs.angularjs.org/api/ng/service/$http).

We should use the 'headers` property to change http headers.